packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -51,6 +51,12 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
      */
     getLongName(element: URI | URIIconReference | FileStat): string | undefined {
         const uri = this.getUri(element);
+        if (uri) {
+            const formatting = this.findFormatting(uri);
+            if (formatting) {
+                return this.formatUri(uri, formatting);
+            }
+        }
         const relativePath = uri && this.workspaceVariable.getWorkspaceRelativePath(uri);
```

Comment:
so this code is never called anymore? We should then replace it with proper formatters that behaviour does not regress compare to master.

Reply:
It is used in case when no suitable formatters was found.